src/constants.jl

```diff
@@ -28,3 +29,6 @@ const DAYS_PER_YEAR        = 365.25
 const DAYS_PER_CENTURY     = 365.25 * 100.0
 
 const YEARS_PER_CENTURY    = 100.0
+
+# ============================= ERFA CONSTANTS =========================
```

Comment:
Nitpick: While you ported this constant from ERFA, it is not ERFA-specific. It is a general (astronomical) time system constant.

Just remove the comment 😉